### PR TITLE
Rework cludo ssh key config, etc.

### LINKED ISSUE #52: Support simpler `cludo.yaml` client configuration
Current `~/.cludo/cludo.yaml` format:

```yaml
client:
  key_path: "~/.ssh/my_ssh_key"
  shell_path: "/usr/local/bin/bash"
```

Desired:

``` yaml
ssh_key_paths: ["~/.ssh/so_cludo", "~/.ssh/another_cludo"]
```

The README has already been updated to match this new format (which means it's _currently_ incorrect).

I _think_ that this may also require a new feature (supporting multiple client SSH keys for authentication).  If so, and assuming that's not trivial:

1. break that into a new issue
2. add a validation that causes the CLI to throw an error if this array has more than one key "We don't yet support multiple SSH keys for authentication.  See URL_TO_ISSUE"

### PR DESCRIPTION
**Pull request by spkane**
Fixes #52 

and a few other minor things along the way...

### CODE REVIEW

**Comment by spkane** [comment]
```
$ grep ssh_ ~/.cludo/cludo.yaml
ssh_key_paths: ["~/.ssh/id_rsa_aws_2020", "~/.ssh/id_ed25519_2020"]

$ aws ec2 describe-instances

Unable to locate credentials. You can configure credentials by running "aws configure".

$ ./builds/darwin_amd64_cludo --debug shell
[WARN] Currently cludo only uses the first SSH key in the config list.
See: https://github.com/superorbital/cludo/issues/81

$ aws ec2 describe-instances | head -n 5
- Reservations:
  - Groups: []
    Instances:
    - AmiLaunchIndex: 0
      Architecture: x86_64
```

File: cludo/root.go [hunk 1]
```diff
@@ -36,8 +36,8 @@ func MakeRootCmd(exit func(int)) (*cobra.Command, error) {
 	viper.BindPFlag("target", rootCmd.PersistentFlags().Lookup("target"))
 	rootCmd.PersistentFlags().Bool("interactive", true, "Should the CLI prompt the user for input")
 	viper.BindPFlag("client.default.interactive", rootCmd.PersistentFlags().Lookup("interactive"))
-	rootCmd.PersistentFlags().StringP("key-path", "k", "~/.ssh/id_rsa", "Path to SSH private key")
-	viper.BindPFlag("client.key_path", rootCmd.PersistentFlags().Lookup("key-path"))
+	rootCmd.PersistentFlags().StringSliceP("ssh-key-paths", "k", []string{"~/.ssh/id_rsa"}, "A comma seperated list of SSH private key filesystem paths")
```
Comment (tammersaleh) [inline cludo/root.go:39]: Is"~/.ssh/id_rsa" the default or used as an example?  I don't think we want to include a default here.

**Review by tammersaleh — CHANGES_REQUESTED** [review]

File: cludo/root.go (hunk 1 above)
Comment (spkane) [reply cludo/root.go:39]: `~/.ssh/id_rsa` is the default for `cludo` and also matches the default key name generate by `ssh-keygen`.

```sh
$ ssh-keygen
Generating public/private rsa key pair.
Enter file in which to save the key (/Users/spkane/.ssh/id_rsa):
```

We can certainly remove it if you'd really prefer. I'd be curious to know what you're concerned about.

Also, it is worth noting that RSA is currently the only key type that is supported by `cludo` at the moment.

Comment (tammersaleh) [reply cludo/root.go:39]: Forgot that RSA is the only format - thanks for reminding me.

`id_rsa` is the default key for `ssh-keygen`, but having cludo auto-find that feels a little too clever.  I'd rather that we remove any defaults, but give an example `.cludo.yaml` that points at that key.

Comment (tammersaleh) [reply cludo/root.go:39]: HOWEVER - that's not part of this PR.  merging.

**Pull request merged**